packages/create-codes/src/index.ts

```diff
@@ -217,6 +217,8 @@ function copyCommon(appDir: string, sharedConfigDir: string) {
   );
 
   // Rewrite package.json
+  packageObjs.name = appDir.replace(/.*\//, "");
```

Comment:
> BTW, I would like to implement it someday. but currently, there is no established ci testing flow to verify if the generated template is functioning correctly.

Yes, I had the same thought when I was cleaning the script. 😅 
I think this we should consider this a high priority task

Reply:
> I think it might be a good to let users determine the package name from the directory they specify, considering the common usage.

In the future, it may be better to add an additional prompt in the script to ask the user to specify the package name. There may be cases where the directory name and the package name are not the same.